app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx

```diff
@@ -24,6 +25,7 @@ interface IContainerState {
   dockerImages: IEcsDockerImage[];
   targetGroupsAvailable: string[];
   targetGroupMappings: IEcsTargetGroupMapping[];
+  containerMappings: IEcsContainerMapping[];
```

Comment:
Now that I think about it, you shouldn't need to set state here. Container mappings is not modified by Container inputs. What you should do is actually set cmd.containerMappings to null or an empty array of ContainerMappings. ContainerMappings are only used when taskDefArtifacts are used.